res/translations/messages.en.po

```diff
@@ -2649,7 +2649,13 @@ msgid "error201"
 msgstr "\"$unit($unit) in $region($region): '$order($command)' - Race and target unit have not been supplied.\""
 
 msgid "newbieimmunity"
-msgstr "\"Your faction is immune against assaults for $int($turns) more weeks.\""
+msgstr "\"Your faction will be immune against assaults for the next $int($turns) weeks.\""
+
+msgid "newbieimmunityending"
+msgstr "\"Your faction will be immune against assaults only one more week.\""
```

Comment:
"assaults" ist ein sehr starkes Wort, da denke ich an SWAT Teams und Panzer. Schlage vor, dass wir "attacks" schreiben, denn es geht ja (primär) um den ATTACKE Befehl.

Reply:
Du hast recht, aber gerade weil es nicht nur um ATTACKIERE geht, ist das auch schwierig. "Protected from attacks" würde ich deshalb schreiben. "Protection from some forms of attack" wäre auch richtig, aber überkorrekt.